e2e/custom.ini

```diff
@@ -0,0 +1,2 @@
+[feature_toggles]
+enable = live-service-web-worker
```

Comment:
This means that all e2e tests will run with this feature toggle now, correct? Can we make this run only for benchmarks?

Reply:
correct, but I shouldn't have included this file in the PR - I used it just for comparison purposes. removed